recipe/meta.yaml

```diff
@@ -18,11 +18,11 @@ requirements:
     build:
         - python  # [win]
         - cmake
-        - libtiff 4.0.6
+        - libtiff 4.0.*
```

Comment:
What were you seeing? Was it this https://github.com/ContinuumIO/anaconda-issues/issues/762?